shared/login/forms/container.desktop.js.flow

```diff
@@ -0,0 +1,9 @@
+/* @flow */
+import React from 'react'
+
+export type Props = {
+    onBack: () => void
+};
+
```

Comment:
You can still declare the export a `React.Component` and you'll get the same type checking.

i.e.
```
declare export default class Container extends React.Component {
  props: Props;
}

```